### ISSUE TITLE
Latex mode bug

### ISSUE THREAD

**Issue by nibage** [plotly/plotly.js#559]
In the example at https://plot.ly/javascript/LaTeX/ when hovering over the plotted points, the graph shows latex code instead of rendered math text.

**Comment by mdtusz** [comment]
Confirmed as a bug. Thanks for reporting this!.

**Comment by etpinard** [comment]
Yep, thanks for reporting. This has been a _bug_ ever since we added support for MathJax more than two years ago.

**Comment by etpinard** [comment]
It might worthwhile to look into swapping MathJax for [KaTeX](https://github.com/Khan/KaTeX).

**Comment by yankev** [comment]
It seems like the plot fails to generate when you include two $ in some text field that needs to be rendered. 
https://jsfiddle.net/60eqc0xz/2/

**Comment by spencerlyon2** [comment]
@etpinard has any testing been done on moving to KaTeX?

**Comment by etpinard** [comment]
@spencerlyon2 No. This is not a high priority for us at the moment,

**Comment by archmoj** [comment]
> It might worthwhile to look into swapping MathJax for [KaTeX](https://github.com/Khan/KaTeX).

The following page could be helpful for comparing MathJax v3 and KaTeX:
https://news.ycombinator.com/item?id=22582343

**Comment by ChasingZenith** [comment]
From my browser, it look like this, maybe escape sequence prefix is needed, for example `'\\alpha'`
 instead of `'\alpha'`

#### on Chrome
![Screenshot_20220913_210136](https://user-images.githubusercontent.com/22112456/189907764-76f8c363-a631-45eb-b91a-861c5455af95.png)

Also, I find that on firefox, the legend size is not correct. (Maybe I should open another issue.)

#### On firefox
![Screenshot_20220913_210346](https://user-images.githubusercontent.com/22112456/189908258-a150106d-a1a4-4d36-a7c5-353f3a0ff3c2.png)

Another example for incorrect legend in firefox

#### on firefox


![Screenshot_20220913_210715](https://user-images.githubusercontent.com/22112456/189909269-3f3b2d70-0cd2-44b4-9a01-950929eba9aa.png)

#### on chrome

![Screenshot_20220913_210847](https://user-images.githubusercontent.com/22112456/189909420-c1013609-5efe-45fa-85cb-4de9fd2dbd82.png)

**Comment by khumarahn** [comment]
Now latex doesn't render in the basic example here: https://plotly.com/javascript/LaTeX/
![image](https://github.com/user-attachments/assets/3def4d41-a401-40a4-99e7-404b97334dc8)
